.github/workflows/issue-labelert.yml

```diff
@@ -0,0 +1,38 @@
+name: Issue labeler
```

Comment:
Initial comment explaning the purpose of this file. It's not trivial.

Reply:
Fix typo in file name.

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1379/commits/e1c9404d075cdc9470ca84ac121df335930d891e